docs/usage/from-a-terminal.rst

```diff
@@ -747,14 +911,26 @@ Want as less as possible data on screen? This argument returns as less as
 possible on screen!
 
 
+------
+
 Multiprocessing
 ^^^^^^^^^^^^^^^
 
 :code:`-w` | :code:`--max-workers`
 """"""""""""""""""""""""""""""""""
 
+    .. versionadded:: 4.0.0
+
 Sets the number of maximal worker to use.
 
+Keep in mind that the :code:`--max-workers` mostly - if not only - affects
+the tester threads. Because we want to safely write the files, we still
+need a single thread which read the submitted results and generate the
+outputs.
+
+The reason we added this to PyFunceble :code:`4.0.0` is we don't want to
+have a wrongly formatted output file.
+
 The default is number of available CPU cores multiplied by 5
```

Comment:
Mhh.... I changed that behavior because it was good with the thread. But not with processes. Meaning that as of now.

If you have more than 2 CPUs the default will be `CPU - 2`. Otherwise, it will `1`.